pyaedt/modeler/circuits/PrimitivesCircuit.py

```diff
@@ -375,6 +374,27 @@ def create_model_from_touchstone(self, touchstone_full_path, model_name=None):
         >>> oModelManager.Add
         >>> oComponentManager.Add
         """
+
+        def _parse_ports_name(file):
+            """Parse and interpret the option line in the touchstone file
+            Parameters
+            ----------
+            file : str
+                Path of the touchstone file.
```

Comment:
A quick search showed that it's a propriety file format and should start with a capital letter: "Touchstone"